index.js

```diff
@@ -1,30 +1,36 @@
-const IncomingWebhook = require('@slack/client').IncomingWebhook;
+const { IncomingWebhook } = require('@slack/client');
 const humanizeDuration = require('humanize-duration');
 const config = require('./config.json');
 
+const token = process.env.GITHUB_TOKEN;
+const { getCommitAuthor } = require('./lib/getCommitAuthor');
+
 module.exports.webhook = new IncomingWebhook(config.SLACK_WEBHOOK_URL);
 module.exports.status = config.GC_SLACK_STATUS;
 
 // subscribe is the main function called by GCF.
-module.exports.subscribe = (event, callback) => {
-  const build = module.exports.eventToBuild(event.data.data);
+module.exports.subscribe = async (event) => {
+  const build = module.exports.eventToBuild(event.data);
 
   // Skip if the current status is not in the status list.
   const status = module.exports.status || ['SUCCESS', 'FAILURE', 'INTERNAL_ERROR', 'TIMEOUT'];
   if (status.indexOf(build.status) === -1) {
-    return callback();
+    return;
   }
 
   // Send message to slack.
-  const message = module.exports.createSlackMessage(build);
-  module.exports.webhook.send(message, (err, res) => {
-    if (err) console.log('Error:', err);
-    callback(err);
+  const message = await module.exports.createSlackMessage(build);
+  module.exports.webhook.send(message, async (err, res) => {
+    if (err) {
+      console.log('Error:', err);
```

Comment:
The error should be surfaced to the caller

Reply:
refactored code, please review